### PR TITLE
Use plogr

### PR DESCRIPTION
**Pull request by krlmlr**
to simplify debugging.

### CODE REVIEW

**Comment by krlmlr** [comment]
Some of these `LOG_DEBUG` could be promoted to `LOG_VERB` or similar, but this is a start.

File: src/collector-method.cpp [hunk 1]
```diff
@@ -145,11 +165,15 @@ class Collector_Scalar : public Collector_Scalar_Base {
...
+
     this->data = Rf_allocVector(VECSXP, length);
     this->current_row = 0;
   }
 
   inline void add_value(SEXP value, Path& path) {
+    LOG_DEBUG;
+
     if (Rf_isNull(value)) {
       SET_VECTOR_ELT(this->data, this->current_row++, this->na);
```
Comment (mgirlich) [inline src/collector-method.cpp:178]: Maybe add a logging statement for the `NULL` case?

File: src/collector-method.cpp [hunk 2]
```diff
@@ -612,6 +710,8 @@ class Multi_Collector {
   int ind[INDEX_SIZE];
 
   inline bool have_fields_changed(SEXP field_names, const int& n_fields) const {
+    LOG_DEBUG << n_fields;
```
Comment (mgirlich) [inline src/collector-method.cpp:713]: Not sure I would expect `have_fields_changed()` to output the number of fields. Maybe add something like `# fields` before?

**Review by mgirlich — APPROVED** [review]

**Comment by mgirlich** [comment]
Thanks. The logging is great! This would have been useful a couple of times...

**Pull request merged**